pytest.ini

```diff
@@ -4,7 +4,8 @@ filterwarnings =
     ignore:No GPU/TPU found, falling back to CPU.:UserWarning
     ignore:Explicitly requested dtype.*is not available.*:UserWarning
     ignore:jax.experimental.vectorize is deprecated.*:FutureWarning
-    # The rest are for experimental/jax2tf
+    ignore:Reductions won't accept lists and tuples*:FutureWarning
```

Comment:
Wouldn't it perhaps be better to fix whatever code in the tests is triggering the warning?

(If there's a particular test that tests the current list/tuple behavior, I would use `jax.test_util.ignore_warnings` around that test alone.)

Reply:
Frankly I don't understand much of the code in jax2tf, not even exactly where is it and how it affects the tests, and probably wouldn't know how to fix what's going on all by myself.
If someone could point me towards where is the failing test and maybe some general idea of what jax2tf is doing I could try to fix the code